channels/chan_audiosocket.c

```diff
@@ -275,6 +275,8 @@ static struct ast_channel *audiosocket_request(const char *type,
 		goto failure;
 	}
 	ast_channel_set_fd(chan, 0, fd);
+	/* Add this line to prevent device state caching for ephemeral channel */
```

Comment:
This comment is weird. It should just be "Prevent device state caching as this channel involves ephemeral destinations or sources".

Or no comment at all